libp2p/stream/lpstream.nim

```diff
@@ -246,23 +246,20 @@ method write*(
   raiseAssert("[LPStream.write] abstract method not implemented!")
 
 method writeLp*(
-    s: LPStream, msg: openArray[byte]
-): Future[void] {.
-    base, async: (raises: [CancelledError, LPStreamError], raw: true), public
-.} =
+    s: LPStream, msg: seq[byte]
+) {.base, async: (raises: [LPStreamError, CancelledError]), public.} =
```

Comment:
this function should not be changed since doing so introduces an additional copy of all data that flows through libp2p, effectively - it would a a significant regression on the hot path.